### ISSUE TITLE
Incorrect documentation for post-processing inline forms

### ISSUE THREAD

**Issue by lapseofreason** [flask-admin/flask-admin#1738]
To post-process inline forms, the documentation for [flask_admin.contrib.sqla.ModelView.inline_model_form_converter](https://flask-admin.readthedocs.io/en/latest/api/mod_contrib_sqla/#flask_admin.contrib.sqla.ModelView.inline_model_form_converter) states to override `post_process` in `InlineModelConverter`. However, doing so has no effect.

I tried it with `postprocess_form` instead of `post_process`, but this did not have any effect either. What is the correct function to override for post-processing inline forms?

**Comment by mpnordland** [comment]
@lapseofreason I don't know what's up with the docs, but you can override the contribute function in your class to do what ever customization you need to do.

**Comment by lapseofreason** [comment]
Thanks, do you mean in flask_admin.contrib.sqla.form.InlineModelConverter? I will try it, but it seems like a bit much as it does  a lot of stuff and it would be necessary to keep it updated with every new version of flask-admin.

**Comment by mpnordland** [comment]
Yes, you can use super() to call the parent method and then modify the result. That should allow you to maintain compatibility. 


---- On Sat, 25 May 2019 14:15:18 -0400 notifications@github.com wrote ----



Thanks, do you mean in flask_admin.contrib.sqla.form.InlineModelConverter? I will try it, but it seems like a bit much as it does a lot of stuff and it would be necessary to keep it updated with every new version of flask-admin.

**Comment by jbbqqf** [comment]
Hi! Triaging older issues -- I opened **#2901** to address this one.

Evidence:
- The `inline_model_form_converter` docstrings in [`flask_admin/contrib/sqla/view.py:213-224`](https://github.com/pallets-eco/flask-admin/blob/master/flask_admin/contrib/sqla/view.py#L213-L224) and [`flask_admin/contrib/peewee/view.py:111-123`](https://github.com/pallets-eco/flask-admin/blob/master/flask_admin/contrib/peewee/view.py#L111-L123) showed a `post_process` method on an `InlineModelConverter` subclass. That method name is never invoked anywhere in the codebase, which is the cause of the original "doing so has no effect" report.
- The hook actually called by the SQLA converter is `InlineFormAdmin.postprocess_form` (`flask_admin/contrib/sqla/form.py:1014,1139`) -- which is what the shipped example at [`examples/sqla_custom_inline_forms/main.py:132`](https://github.com/pallets-eco/flask-admin/blob/master/examples/sqla_custom_inline_forms/main.py#L132) already uses.
- On the Peewee backend, `postprocess_form` *also* wasn't being called by `InlineModelConverter.contribute`, so even the working SQLA pattern was silently a no-op there. The PR fixes both: corrects the docstrings and wires `info.postprocess_form(child_form)` into the Peewee `contribute()` path. Regression tests added in `flask_admin/tests/sqla/test_inlineform.py` and `flask_admin/tests/peeweemodel/test_basic.py`.

Posting here so you can follow #2901 if you'd like to weigh in / get notified when it merges.

---
*Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the references above were verified manually.*
